templates/data/opensearch/what-is-opensearch.html

```diff
@@ -0,0 +1,582 @@
+                        <a href="https://opensearch.org/docs/latest/replication-plugin/index/">Cross-cluster replication</a> – Replicate your data across multiple OpenSearch clusters
+                    </li>
+                </ul>
+
+            </div>
+        </div>
+    </section>
+
+    <section class="p-section">
+        <div class="row--50-50 p-section--shallow">
+            <hr />
+            <div class="col">
+                <h2>Canonical’s Charmed OpenSearch</h2>
+            </div>
+            <div class="col">
+                <p>
+                    Secure and automate the deployment, maintenance and upgrades of your search and analytics suite across private and public clouds.
+                </p>
+                <hr class="is-muted" />
+                <p>
+                    <a class="p-button--positive"  href="/data/opensearch#get-in-touch">Contact Us</a>
+                    <a class="p-button"
+                       href="https://assets.ubuntu.com/v1/6dc3440a-Charmed%20OpenSearch%20-%20Datasheet%20-%20amended%2004%2006%202024.pdf?_gl=1*76e20f*_gcl_au*MTY4OTkyMTA1My4xNzE2NTQxNjg0">Datasheet</a>
+                </p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-9 col-medium-6 col-start-large-4">
+                <div class="row">
+                    <hr class="is-muted" />
+                    <div class="col-3">
+                        <h3 class="p-heading--5">Charmed OpenSearch</h3>
+                    </div>
+                    <div class="col-6">
+                        <p class="p-heading--5">Included in Ubuntu Pro + Support</p>
+                        <p>When you purchase an Ubuntu Pro + Support plan, you also get support for the full Charmed OpenSearch solution.</p>
+                        <hr class="is-muted" />
+                        <div class="p-section--shallow">
+                            <ul class="p-list--divided">
+                                <li class="p-list__item is-ticked">Up to 10 years of OpenSearch support per release track</li>
+                                <li class="p-list__item is-ticked">24/7 or weekday phone and ticket support</li>
+                                <li class="p-list__item is-ticked">
+                                    Up to 10 years of security maintenance for OpenSearch covering critical and high severity CVEs
+                                </li>
+                            </ul>
+                            <hr class="is-muted" />
+                            <p>
+                                Charmed OpenSearch allows you to automate deployment and operation of OpenSearch at web scale in the environment of your choice – on the cloud or in your data centre.
+                            </p>
+                            <hr class="is-muted" />
+                            <p>
+                                <a href="/data/OpenSearch/managed">Explore Managed OpenSearch Solutions&nbsp;&rsaquo;</a>
```

Comment:
This link 404s

Reply:
This page is still in development and hence this branch merges to feature branch.
Once all the pages are made this will be merged to main.